src/database/schema/ChannelListSchema.ts

```diff
@@ -373,7 +380,11 @@ class ChannelList implements BaseDbSchema {
       lastUpdatedBy: json?.message?.user?.id,
       createdAt: json?.channel?.created_at,
       rawJson: json,
-      user: null
+      user: null,
+      anon_user_info_color_code: json?.anon_user_info_color_code,
+      anon_user_info_color_name: json?.anon_user_info_color_name,
+      anon_user_info_emoji_name: json?.anon_user_info_emoji_name,
+      anon_user_info_emoji_code: json?.anon_user_info_emoji_code
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

New properties related to anonymous user information have been added to the object. However, it seems like these properties are directly accessed from `json` without any null checks. If `json` is null or undefined, this could lead to a runtime error. Consider adding null checks before accessing these properties.
```diff
- anon_user_info_color_code: json?.anon_user_info_color_code,
- anon_user_info_color_name: json?.anon_user_info_color_name,
- anon_user_info_emoji_name: json?.anon_user_info_emoji_name,
- anon_user_info_emoji_code: json?.anon_user_info_emoji_code
+ anon_user_info_color_code: json ? json.anon_user_info_color_code : null,
+ anon_user_info_color_name: json ? json.anon_user_info_color_name : null,
+ anon_user_info_emoji_name: json ? json.anon_user_info_emoji_name : null,
+ anon_user_info_emoji_code: json ? json.anon_user_info_emoji_code : null
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->